docs/contributing/testing.md

```diff
@@ -0,0 +1,112 @@
+# Test your GOV.UK Frontend contribution
+
+Before you submit your contribution, you should:
+
+- check your changes look right and behave correctly
+- test in supported browsers and assistive technology
+- run our automated tests, and update tests or add new tests if our tests fail
+
+[Contact us](https://design-system.service.gov.uk/#support) for help if either:
```

Comment:
Perhaps a more appropriate place to ask for help is when they raise the pull request, rather than a separate channel.

We could consider saying, something like:

> If you need help doing any of this testing let us know in your pull request or (contact us](...)

sorta thing, we don't want emails if we can have discussions on pull requests I think